.github/workflows/deploy.yml

```
@@ -3,45 +3,113 @@ on:
-          -av --delete target/ ${SSH_USER}@${SSH_HOST}:${DST_PATH}
+          -av --delete deploy/ ${SSH_USER}@${SSH_HOST}:${DST_PATH}
         env:
           SSH_USER: ${{ secrets.SSH_USER }}
           SSH_HOST: ${{ secrets.SSH_HOST }}
           DST_PATH: ${{ secrets.DST_PATH }}
-      - name: Deploy secrets
+
+      # 環境変数の設定
+      - name: Setup environment variables
         run: >
           ssh -i ${{ runner.temp }}/key -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null ${SSH_USER}@${SSH_HOST} 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)